proto/pdpb.proto

```diff
@@ -668,3 +679,36 @@ message GetDCLocationInfoResponse {
     // which it's set when the number of the max suffix bits changes.
     Timestamp max_ts = 3;
 }
+
+message GetAllKeyspacesRequest {
+    RequestHeader header = 1;
+}
+
+message GetAllKeyspacesResponse {
+    ResponseHeader header = 1;
+    uint64 keyspace_version = 2;
+    // All registered keyspaces.
+    repeated keyspace.Keyspace keyspaces = 3;
+    // Prefixes that were registered to a keyspace and then unbound from it.
+    repeated keyspace.KeyspacePrefix unbound_prefixes = 4;
```

Comment:
why need this?

Reply:
Because `KeyspacePrefix` can't be removed. So it'll leave history even if no keyspace refers to it. With `unbound_prefixes` the users get to know these kind of prefixes and then re-bound it to a keyspace.